configuration/concepts/commons.csv

```diff
@@ -89,13 +89,14 @@ ca69545b-dfef-4210-b311-dc8a1d309311,,CIEL:540,Anterior,Antérieur,Anterior,Ant
 f5c721f6-7162-4533-9a83-4a4cd6997cb8,,CIEL:542,Lateral,Lateral,Lateral,Lateral,,,Misc,N/A,,,,,,,,,,,
 4d1f668f-f9b7-4e95-8436-53f37e8c643e,,,Other observations,Autres observations,Other observations,Autres observations,,,Misc,Text,,,,,,,,,,,
 b0823b2c-4bc8-2a7c-41b5-7bf3c7f14e82,,,Duration,Durée,Duration,Durée,,,Misc,Numeric,,,,,,,,,,,
-88e5cc3c-0ba4-11eb-a3aa-0242ac140008,,CIEL:1074,Month(s),Moi(s),Month(s),Moi(s),,,Misc,N/A,,,,,,,,,,,
+88e5cc3c-0ba4-11eb-a3aa-0242ac140008,,CIEL:1074;SNOMED:258706009,Month(s),Moi(s),Month(s),Moi(s),,,Misc,N/A,,,,,,,,,,,
 a4d0c480-923b-49fe-b125-80453675fdea,,CIEL:1734,Year(s),An(s),Year(s),An(s),,,Misc,N/A,,,,,,,,,,,
 88e474ea-0ba4-11eb-a3aa-0242ac140008,,CIEL:1822,Hour(s),Heure(s),Hour(s),Heure(s),,,Misc,N/A,,,,,,,,,,,
-327cdb97-571c-2d13-306b-4024fce268cb,,CIEL:1072,Day(s),Jour(s),Day(s),Jour(s),,,Misc,N/A,,,,,,,,,,,
-88e55059-0ba4-11eb-a3aa-0242ac140008,,CIEL:1073,Week(s),Semaine(s),Week(s),Semaine(s),,,Misc,N/A,,,,,,,,,,,
+327cdb97-571c-2d13-306b-4024fce268cb,,CIEL:1072;SNOMED:258703001,Day(s),Jour(s),Day(s),Jour(s),,,Misc,N/A,,,,,,,,,,,
+88e55059-0ba4-11eb-a3aa-0242ac140008,,CIEL:1073;SNOMED:258705008,Week(s),Semaine(s),Week(s),Semaine(s),,,Misc,N/A,,,,,,,,,,,
 ae703c82-ed81-4b14-be4b-966949a1a696,,,Acute,Aigu,Acute,Aigu,,,Misc,N/A,,,,,,,,,,,
 9de275a5-d92c-4868-8430-973ebcc9d593,,,Chronic,Chronique,Chronic,Chronique,,,Misc,N/A,,,,,,,,,,,
+b1efceca-31e0-4369-9dc1-73f483ba5b79,,,Duration Units,Unités de durée,Duration Units,Unités de durée,,,Question,Coded,,Day(s);Week(s);Month(s),Day(s);Week(s);Month(s),,,,,,,,
```

Comment:
`Duration Units` and `Time Units` seems to describe the same thing.
Either we need to name it as "Medication Duration Units" or we merge them and use only one concept for both.

Would that be possible?

Reply:
Since they are describing different things and have different datatypes we should only rename it to Medication Duration Units as proposed...Could you provide the French locale name

Reply:
>they are describing different things

Are they? I think they describe the same thing: "units of time"
>have different datatypes

So then no choice but to rename and make it clearer that this is a special set for medication durations.
In French that'd be "Unités de durée de traitement".
I guess in English that could be "Treatment duration units"